### DISCUSSION TITLE
cannot change the `min-height` of `collapse-title` to `0px`

### DISCUSSION THREAD

**Discussion by Rafi-Riday** [saadeghi/daisyui#1794] [Q&A] [answered]
I'm trying to make some custom design, but I cannot change the `min-height` of `collapse-title` class to `0px` . I need to do it for my design. I tried to use syle attribute, i also put !important, tried to @apply min-h-0, but nothing worked;
But in chrome dev tool, under `collapse-title` class, if i deselect te `min-height: 3.75rem;`, it works then.

**Comment by saadeghi** [comment]
min-height in CSS is the minimum height so if you want the height to be `0`, you should also set `h-0`  
Also in daisyUI `collapse-title` has a padding.
  
So you need to add these 3 classes: `h-0 min-h-0 p-0`  
https://play.tailwindcss.com/NfudPZn06F

**Reply by Rafi-Riday** [reply]
Still not working, the title is taking the same height as min-height: 3.75rem;

code :
```
<section class="collapse collapse-plus">
    <input type="checkbox" class="peer border h-0" />
    <h2 class="text-2xl font-medium collapse-title p-0 h-0 min-h-0 border">
        Title Text Here
    </h2>
    <div class="collapse-content">
        Body here
    </div>
</section>
```
Here i just want to make the "Title Text Here" to not take that much space. I want to remove other height, padding rather than the text height

**Comment by Rafi-Riday** [comment, marked as answer]
Solve :
```
<section class="collapse collapse-plus">
    <input type="checkbox" class="peer min-h-0" />
    <h2 class="text-2xl font-medium collapse-title p-0 min-h-0 border">
        Title Text Here
    </h2>
    <div class="collapse-content">Body here</div>
</section>
```
If using `<input type="checkbox" class="peer" />` then have to put `min-h-0` in this `peer` class also
